@c-w CI run didn't finish yet, but locally tests seem to be failing with:

```bash

________________________________________________________________ ERROR at setup of MinioTest.test_objects_stream_iterable _________________________________________________________________

cls = <class 'integration.storage.test_minio.MinioTest'>

>   ???
E   ValueError: Container fff2b0bdc3 failed to start and become ready in 30 seconds (did not find "IAM initialization complete" message in container logs).
E   
E   Container Logs:
E   API: http://172.17.0.2:9000  http://127.0.0.1:9000 
E   
E   Console: http://172.17.0.2:42801 http://127.0.0.1:42801 
E   
E   Documentation: https://docs.min.io
E   
E   WARNING: Console endpoint is listening on a dynamic port (42801), please use --console-address ":PORT" to choose a static port.
```

I didn't dig in much yet. Do you think that's related to the upstream change in the Docker image (in this case we should pin it to a known working tag and to that + update ready message) or something else?

EDIT: CI run completed, looks like it contains the same error - https://github.com/apache/libcloud/runs/3976839689?check_suite_focus=true.

--- a/integration/storage/base.py
+++ b/integration/storage/base.py
@@ -322,7 +322,7 @@ class Integration:
         container = None
         verbose = False
 
-        container_ready_timeout = 30
+        container_ready_timeout = 20
 
         @classmethod
         def setUpClass(cls):
@@ -350,7 +350,6 @@ class Integration:
 
             # We register atexit handler to ensure container is always killed, even if for some
             # reason tearDownClass is sometimes not called (happened locally a couple of times)
-
             atexit.register(cls._kill_container)
 
             wait_for(cls.port, cls.host)

--- a/integration/storage/test_minio.py
+++ b/integration/storage/test_minio.py
@@ -30,7 +30,7 @@ class MinioTest(Integration.ContainerTestBase):
     port = 9000
     environment = {'MINIO_ROOT_USER': account, 'MINIO_ROOT_PASSWORD': secret}
     command = ['server', '/data']
-    ready_message = b'IAM initialization complete'
+    ready_message = b'Console endpoint is listening on a dynamic port'
 
     def test_cdn_url(self):
         self.skipTest('Not implemented in driver')

